Modules/_ctypes/libffi_osx/include/ffi.h

```diff
@@ -274,7 +274,7 @@ typedef struct ffi_raw_closure {
 #if !FFI_NATIVE_RAW_API
 	/*	if this is enabled, then a raw closure has the same layout 
 		as a regular closure.  We use this to install an intermediate 
-		handler to do the transaltion, void** -> ffi_raw*. */
+		handler to do the translation, void** -> ffi_raw*. */
```

Comment:
We should avoid applying changes to the /libffi_osx/ subtree as that is a copy of a third party open source project.  (revert this file and the next couple of files)